### PR TITLE
refine GNSS helpsheet

### PR DESCRIPTION
**Pull request by elidana**
Hi,

this is a modified version of the GNSS helper sheet Mark initially wrote. I have been using it for a while now as a checklist for delta changes.

### CODE REVIEW

File: docs/sheets/GNSS.md [hunk 1]
```diff
@@ -1,236 +1,251 @@
...
+* :scroll: :six: [__creating updating or closing__ a GNSS site](#openclose) :dragon:
+
 
 Global Navigation Satellite System (GNSS) equipment are installed at GNSS __marks__. Each __mark__ is
 attached to a __monument__. The details of the __mark__, as found in the __network/marks.csv__ file,
-tend to relate more to documentation, such as name and location. Whereas the __network/monuments.csv__ file
+tend to relate more to documentation, such as mark identification and location. Whereas the __network/monuments.csv__ file
 tends to list physical aspects of the actual monument as constructed.
 
 Equipment installed should be found in the the relevant __assets__ directory files. Actual times when
```
Comment (staylorofford) [inline docs/sheets/GNSS.md]: "Equipment installed should be found in the the relevant" here "the" is repeated

File: docs/sheets/GNSS.md [hunk 2]
```diff
@@ -1,236 +1,251 @@
...
 __antennas__ and __receivers__ are deployed is managed via the __install/antennas.csv__ and __install/receiver.csv__
 files respectively. As the firmware of the __receiver__ is important for data post-processing, this is maintained
 in the __install/firmware.csv__ file. This is independent of installation and can also be maintained for instruments
-not actively deployed.
+not actively deployed. __radomes__ and __metsensors__ are occasionally installed at GNSS marks.
```
Comment (staylorofford) [inline docs/sheets/GNSS.md]: should there be a mention of the corresponding files like above?

File: docs/sheets/GNSS.md [hunk 3]
```diff
@@ -1,236 +1,251 @@
...
-## :one: <a name="mark"></a>_Creating or updating a GNSS mark_
+A complete list of files that cover GNSS metadata is below.
 
-For new marks a __code__ will need to be assigned by the appropriate mechanism.
+folder | tables
+-------|-------
+assets | antennas.csv metsensors.csv radomes.csv receivers.csv
+environment | visibility.csv
+install | antennas.csv firmware.csv metsensors.csv radomes.csv receivers.csv sessions.csv
+network | marks.csv monuments.csv networks.csv
```
Comment (staylorofford) [inline docs/sheets/GNSS.md]: this is a little difficult to read, could we add delimiters such as , or | in the tables column?

File: docs/sheets/GNSS.md [hunk 4]
```diff
@@ -1,236 +1,251 @@
...
 > * Receivers need to be listed in the __assets/receivers.csv__ file.
 > * The associated __mark__ needs to be listed in the __network/marks.csv__ file.
-> 
+>
 > ### :small_orange_diamond: Delta checks
-> 
 > * A given __receiver__ can only be deployed to a single __mark__ at any given time.
 > * A __mark__ can only have one __receiver__ mounted at any given time.
->
+> * An entry in the session file should be matching the receiver settings and covering its deploy time.
```
Comment (staylorofford) [inline docs/sheets/GNSS.md]: should there be a similar requirement for a firmware version?

File: docs/sheets/GNSS.md [hunk 5]
```diff
@@ -1,236 +1,251 @@
...
+> * The operational time range of the __mark__ (Start is the first data in the archive).
+> * The "mark type" of the __monument__.
+> * The ground relationship of the __monument__.
+> * The __monument__ "foundation type".
+> * The __monument__ "foundation depth".
+> * The date the __monument__ was established.
+> * The date the __monument__ was demolished, if applicable.
+>
+> ### :heavy_check_mark: Delta prerequisites
+> * There can only be one __mark__ for any given code.
```
Comment (staylorofford) [inline docs/sheets/GNSS.md]: should this be clearer: for any given mark "code" (same reference style used previously)

File: docs/sheets/GNSS.md (hunk 4 above)
Comment (elidana) [reply docs/sheets/GNSS.md]: this automatic check was never been introduced , so I will remove it for now and will be added once implemented in the codes

**Comment by elidana** [comment]
all changes recommended by @staylorofford have been applied

**Comment by elidana** [comment]
Hi @ozym can I have your thumbs up (or down) for this one please?

**Comment by elidana** [comment]
thanks!

**Pull request merged**